examples/graphql-kotlin/src/main/kotlin/example/armeria/server/graphql/kotlin/Main.kt

```diff
@@ -0,0 +1,67 @@
+package example.armeria.server.graphql.kotlin
+
+import com.expediagroup.graphql.generator.SchemaGeneratorConfig
+import com.expediagroup.graphql.generator.TopLevelObject
+import com.expediagroup.graphql.generator.toSchema
+import com.linecorp.armeria.server.Server
+import com.linecorp.armeria.server.ServerBuilder
+import com.linecorp.armeria.server.graphql.GraphqlService
+import org.slf4j.LoggerFactory
+import kotlin.concurrent.thread
+
+fun main() {
+    val server = newServer(8080)
+    Runtime.getRuntime().addShutdownHook(
+            thread(start = false) {
+                server.stop().join()
+                logger.info("Server has been stopped.")
+            }
+    )
+    server.start().join()
+}
+
+/**
+ * Returns a new {@link Server} instance configured with GraphQL HTTP services.
+ *
+ * @param port the port that the server is to be bound to
+ */
+private fun newServer(port: Int): Server {
+    val sb = Server.builder()
+    sb.http(port)
+    configureService(sb)
+    return sb.build()
+}
+
+fun configureService(sb: ServerBuilder) {
+    //  TODO: GraphQLSchema instance cannot be injected at this time.
```

Comment:
I think I don't get this comment. 😅 
So what do we need to do to solve this problem?

Reply:
For example, we do not provide these methods.

```java
builder().graphqlSchema(toSchema(
    config = SchemaGeneratorConfig(listOf("example.armeria.server.graphql.kotlin")),
    queries = listOf(TopLevelObject(UserQuery()))
)).build()
```

However, I wrote the above comment because we can provide it later.

Reply:
Ah thanks for the explanation. 😉

Reply:
@heowc, How about creating a new issue and adding a link to it in the comment?

Reply:
That's a good idea :)